keyboards/atreus/readme.md

```diff
@@ -18,6 +18,7 @@ If you would like to use one of the alternative controllers:
 
     make atreus/astar:default:avrdude
     make atreus/teensy2:default:teensy
+    make atreus/promicro:default:avrdude
```

Comment:
All of these could be replaced with just: 
```markdown
    make atreus/promicro:default:flash
```

This this reads the `BOOTLOADER` value, and chooses the flasher based on that.